pkg/kubelet/cm/devicemanager/manager.go

```diff
@@ -393,11 +393,8 @@ func (m *ManagerImpl) Allocate(pod *v1.Pod, container *v1.Container) error {
 func (m *ManagerImpl) UpdatePluginResources(node *schedulerframework.NodeInfo, attrs *lifecycle.PodAdmitAttributes) error {
 	pod := attrs.Pod
 
-	m.mutex.Lock()
```

Comment:
Unfortunately I think the `m.sanitizeNodeAllocatable` still needs the lock since it seems to be reading from `m.allocatedDevices`

Reply:
yes, I also forgot about isDevicePluginResource

Reply:
just noting this appears resolved below